app/locales/es_419.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
please ensure that the `_display_name` key is provided in lokalise

Reply:
added it.